python/modules/map_plate.py

```diff
@@ -78,49 +113,129 @@ def _get_param_paths(input_path):
     return param_paths
 
 def _collect_exp_settings(abs_path, mp_file, delimiter, mp_dict):
+    """
+    Function for extracting information about optional 
+    experiment's parameters (i.e. stimulation.1.1), 
+    it parse both name and value of a given parameter
+     for each map_plate well. 
+    """
     rel_path = FM.path_get_relative(abs_path, mp_file)
     fullname = ".".join(FM._path_split(rel_path))
-    extension_length = len(FM.file_get_extension(fullname))  #extracting proper column name from rel_path i.e. compare.1.1 from compare/1.1.csv
+    # Extracting proper column name from rel_path 
+    # i.e. compare.1.1 from compare/1.1.csv
+    extension_length = len(FM.file_get_extension(fullname)) 
     name = fullname[:-extension_length]
     data = CSV_M.read_csv(mp_file, delimiter)
     for well in mp_dict:
-        position = mp_dict[well]["position"]
-        value = (data[position[0]][position[1]])
+        pos_x = int(mp_dict[well]["position_x"])
```

Comment:
Yes, every information in map_plate is saved as a string to allow easy manipulations, easy csv data merging and saving etc. Plus, in fact map_plate is text description of experiment settings. I'm converting those two specific values to (int) because I know what they are and I want to use them as indexes.